<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyDeliveredDoseDialog</class>
 <widget class="QWidget" name="ModifyDeliveredDoseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="dos_deliveredDose_formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="seanceLabel">
      <property name="text">
       <string>Seance</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="seanceComboBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="newDoseLabel">
      <property name="text">
       <string>Nouvelle dose</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="newDoseLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="doseUnitLabel">
      <property name="text">
       <string>Unit√© de dose</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="doseUnitComboBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updatePushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Effectuer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
